src/rs/src/skill.rs

```diff
@@ -331,8 +331,8 @@ pub enum Skill {
 	earthquake(u8),
 	eatspell,
 	elf,
-	embezzle,
-	embezzledeath,
+	embezzle(i16),
+	embezzledeath(i16),
```

Comment:
Better off not making this param, otherwise it just makes for more code generation than necessary

Reply:
Can we make more of an argument than "makes the codebase larger"?

Without parameterization I see coupling between Text and Skill failing Single Responsibility Principle.
Dynamic input enables powerfully chaining skill calls, which benefit by preventing new creation of skill iterations (cross-multiplied by the count of each variable option).

 If embezzle is the only option you see the current behavior, then perhaps I agree.
 
 Please advise,

Reply:
It's easy to add parameters as needed. I'm considering string interning wasm bindgen offers which works best with static strings but there's hope for wasm stringref to come around

Text is inherently coupled with skills. When it comes to principles YAGNI for life

Codebase size is important. I've refactored the whole codebase a few times & that effort scales with code size

Reply:
If size or new dep. is a concern, I believe Serde has a wasm-bindge and your crate already includes Serde core. Might be worth adding to your candidate list.

Reply:
serde is only used as a build dependency